### PR TITLE
Add Organization-Id header support

### PR DESCRIPTION
**Pull request by iamdevice**
RebillyAPI definitions: https://github.com/Rebilly/RebillyAPI/pull/790
RebillyUserAPI definitions: https://github.com/Rebilly/RebillyUserAPI/pull/360
RebillyReportsAPI definitions: https://github.com/Rebilly/RebillyReportsAPI/pull/88

### CODE REVIEW

File: src/Middleware/OrganizationIdHeader.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    /**
+     * Add HTTP header to request.
+     *
+     * {@inheritdoc}
+     */
+    public function __invoke(Request $request, Response $response, callable $next)
+    {
+        if (!is_null($this->organizationId)) {
```
Comment (ashkarpetin) [inline src/Middleware/OrganizationIdHeader.php]:
```suggestion
        if ($this->organizationId) {
```

Comment (ashkarpetin) [reply src/Middleware/OrganizationIdHeader.php]: I didn't find any usages of `is_null`

File: src/Client.php [hunk 2]
```diff
@@ -211,6 +211,10 @@ public function __construct(array $options)
             $sessionToken = null;
         }
 
+        if (!isset($organizationId)) {
+            $organizationId = null;
```
Comment (ashkarpetin) [inline src/Client.php]: Update phpdoc?
https://github.com/Rebilly/rebilly-php/pull/385/files#diff-ff73e042e738204c6da009e2ed19f783L172

**Review by mvphero — APPROVED** [review]

File: src/Middleware/OrganizationIdHeader.php [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    /**
+     * Add HTTP header to request.
+     *
+     * {@inheritdoc}
+     */
+    public function __invoke(Request $request, Response $response, callable $next)
+    {
+        if (isset($this->organizationId)) {
```
Comment (slavcodev) [inline src/Middleware/OrganizationIdHeader.php]: I better would not attach middleware if orhanization is not set.

File: src/Middleware/OrganizationIdHeader.php [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+use Psr\Http\Message\RequestInterface as Request;
+use Psr\Http\Message\ResponseInterface as Response;
+use Rebilly\Middleware;
+
+final class OrganizationIdHeader implements Middleware
+{
+    public const HEADER = 'Organization-Id';
+
+    /** @var string */
```
Comment (slavcodev) [inline src/Middleware/OrganizationIdHeader.php]:
```suggestion
```

**Review by slavcodev — APPROVED** [review]

**Pull request merged**